packages/lesswrong/lib/sql/Query.ts

```diff
@@ -58,6 +58,12 @@ const arithmeticOps = {
   ...comparisonOps,
 };
 
+const variadicFunctions = {
+  $min: "LEAST",
+  $max: "GREATEST",
+  $ifNull: "COALESCE",
+};
```

Comment:
```suggestion
} as const;
```

Yeah, I know, just doing that won't actually make the type of `func` below any better.